cpp/include/cudf/strings/wrap.hpp

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <cudf/strings/strings_column_view.hpp>
+#include <cudf/column/column.hpp>
+
+namespace cudf
+{
+namespace strings
+{
+/**
+ * @brief Inserts new-line characters (ASCII 0x0A) into each string in place of spaces.  
+ *
+ * Returns a column of strings where, for each string row in the input, 
+ * words separated by spaces will become separated by newline characters.
```

Comment:
This implies all the spaces are replaced by new-lines. The new-line is inserted only to meet the `width` requirement.
For example:
```
s = ["the quick brown fox jumped over the lazy brown dog", "hello, world"]
w = wrap(s,12)
w is now ["the quick\nbrown fox\njumped over\nthe lazy\nbrown dog", "hello, world"]
```
If the string is longer than width, a new-line is inserted at space positions so that each line is no more than width characters without truncating any non-space character sequences.

Reply:
Fixed.